Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "geocoding"
 description = "Geocoding library for Rust"
 version = "0.3.1"
-authors = ["Stephan Hügel <urschrei@gmail.com>", "Blake Grotewold <hello@grotewold.me>"]
+authors = ["The Georust Developers <admins@georust.org>"]
```

Comment:
I just tried to contact this address and it bounced:

```
<admins@georust.org>: host gmr-smtp-in.l.google.com[2607:f8b0:400d:c0f::e]
   said: 550-5.1.1 The email account that you tried to reach does not exist.
   Please try 550-5.1.1 double-checking the recipient's email address for
   typos or 550-5.1.1 unnecessary spaces. Learn more at 550 5.1.1
   https://support.google.com/mail/?p=NoSuchUser b14si109472pjg.2 - gsmtp (in
   reply to RCPT TO command)
```

I understand why we might not want to have individual authors listed, but putting in a fake email address seems fraught.

Reply:
It _should_ be working…

Reply:
see https://github.com/georust/meta/issues/28